src/graph/visitor/PropertyTrackerVisitor.cpp

```diff
@@ -193,6 +220,15 @@ void PropertyTrackerVisitor::visit(FunctionCallExpression *expr) {
   }
 }
 
+void PropertyTrackerVisitor::visit(AggregateExpression *expr) {
+  auto funName = expr->name();
+  std::transform(funName.begin(), funName.end(), funName.begin(), ::tolower);
+  if (funName == "count") {
+    return;
+  }
+  expr->arg()->accept(this);
+}
+
```

Comment:
Why return directly?
`count(*)` is different from `count(v.player.age)`.

Reply:
good catch